src/Symfony/Component/DomCrawler/Tests/CrawlerTest.php

```diff
@@ -800,6 +826,18 @@ public function testLinks()
         $this->assertEquals(array(), $this->createTestCrawler()->filterXPath('//ol')->links(), '->links() returns an empty array if the node selection is empty');
     }
 
+    public function testImages()
+    {
+        $crawler = $this->createTestCrawler('http://example.com/bar/')->selectImage('Bar');
+        $this->assertInternalType('array', $crawler->images(), '->images() returns an array');
+
+        $this->assertCount(4, $crawler->images(), '->images() returns an array');
+        $images = $crawler->images();
+        $this->assertInstanceOf('Symfony\\Component\\DomCrawler\\Image', $images[0], '->images() returns an array of Image instances');
```

Comment:
Nice one. Should I do the same with [links](https://github.com/symfony/symfony/blob/6cd873447d5581fd04f382f6c586ab2716ed7d32/src/Symfony/Component/DomCrawler/Tests/CrawlerTest.php#L798)?